### PR TITLE
[*] Command Generate - Fix application host/port usage by asking for it in lumber generate

### PR DESCRIPTION
**Pull request by larcin**

### CODE REVIEW

**Comment by larcin** [comment]
The doc should be updated after the release of this PR (rename APP_HOST/APP_PORT => APPLICATION_HOST/APPLICATION_PORT)

**Comment by larcin** [comment]
[Lumber - APP_PORT/APPLICATION_PORT not taken into account](https://trello.com/c/YdSBAdwj/3591-lumber-appport-applicationport-not-taken-into-account)

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,13 +9,15 @@
...
 - Readme - Remove the Licence section as it it not the right one and it is already accessible in the Github page header.
+- Command Generate - Ask for application host/port when using a connection url.
 
 ### Fixed
 - Command Generate - Fix package name.
 - Command Generate - Remove trailing spaces in `.env`.
 - Technical - Apply ESLint rules to `database-analyzer`.
 - Command Generate - Do not duplicate `createdAt` and `updatedAt` if not named as camelCase in database.
 - Command Generate - Fix generation of field with type array.
+- Command Generate - Fix application host/port usage by asking for it in lumber generate.
```
Comment (VincentMolinie) [inline CHANGELOG.md]: This is not mentioned anywhere that this fix install of lumber with custom port and hostname

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,13 +9,15 @@
...
```
Comment (VincentMolinie) [inline CHANGELOG.md]: This is not a fix and duplicated with the changed

**Pull request merged**